Cargo.lock

```diff
@@ -334,6 +334,15 @@ dependencies = [
  "serde",
 ]
 
+[[package]]
+name = "cadence"
```

Comment:
You mean on `libdatadog_profiling.{a,so}`? The size is exactly the same

Reply:
It won't change the profiling lib. It will change the final package tracer size I assume as the sidecar is used by tracing only for now. Basically, our release artifacts are 500MB already, the profiling team starts looking into reducing package size, we should globally be very careful of the size impact of adding any new dependency.

Reply:
We support 12 versions IIRC. So I assume we have an extra 400KB per version here. Not saying it comes from cadence only though. Anyway, this is something that we will need to be more and more careful about I believe even for the sidecar (not a blocker here I suppose).